addons/crm/views/crm_lead_views.xml

```diff
@@ -1095,8 +1217,69 @@ if record:
             <field name="act_window_id" ref="crm_lead_action_pipeline"/>
         </record>
 
+        <record id="action_opportunity_forecast" model="ir.actions.server">
```

Comment:
Should be located in a new ``data/ir_actions.xml`` file holding server actions, client actions, ...  indeed this is not a view.

Reply:
Should I also move `action_mark_as_lost` and `action_your_pipeline` ?

Reply:
Well could be great. If possible do it in a separate commit ([MOV]).